### PR TITLE
Replace support question template with contact links

### PR DESCRIPTION
**Pull request by Xerkus**
Support question issue template confuses users.
This pull request replaces template with contact links that will be displayed as a button leading to chat or forums instead of creating new issue.

Mirrors laminas/.github#5

### CODE REVIEW

**Review by geerteltink — APPROVED** [review]

**Comment by froschdesign** [comment]
My opinion still stands on this topic: reducing the reference to the chat because important questions and answers are lost after few hours and days in a chat group.

> push the users with questions to the forum, not chat

More informations and background like this can be found in a related pull request on the old maintainer repository: https://github.com/zendframework/maintainers/pull/60

_I myself will not answer any support questions in the chat because I'm not wasting my time writing in a black hole._

**Comment by michalbundyra** [comment]
I agree with @froschdesign that losing chat history is a big problem. Even if we keep the history it will be very hard to search for questions/answers there.

Not sure how it is solved by others (doctrine?)
Not sure also if hiding chat is the best option.

**Comment by froschdesign** [comment]
> Not sure also if hiding chat is the best option.

As said in the past, I do not want to hide or kill the chat because this has its advantages and justification. But for support questions it is a dead end when the history is regularly lost. So pushing the user to the chat is the wrong direction. _(imo)_

**Comment by geerteltink** [comment]
Reading your replies I guess the best option is to merge the two and create a "feedback/support" selection which redirects to discourse?

**Comment by Xerkus** [comment]
Actually it makes  sense. If people are ready to create new issue they can as well create new forum post. There are other ways to discover chat.

**Comment by Xerkus** [comment]
Options are updated to lead to forums, support questions leads directly to questions category

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Support Question
+    url: https://discourse.laminas.dev/c/questions/5
+    about: 'This repository is only for reporting bugs and suggesting features. Support is provided via forums:'
+  - name: 💡 Feedback Wanted
+    url: https://discourse.laminas.dev
+    about: 'Discuss and get feedback on ideas on our forums:'
```
Comment (froschdesign) [inline .github/ISSUE_TEMPLATE/config.yml:8]: Should we extend the sentence to clarify what ideas are meant? Like:

> Discuss and get feedback on ideas for improvements or new features on our forums:

Or would this only be restrictive?

Comment (Xerkus) [reply .github/ISSUE_TEMPLATE/config.yml:8]: Generally, improvement and feature requests would end up as issues I think.
For this I had more broad feedback requests in mind, like those issues where users showcasing library they built on top of the component.

**Review by froschdesign — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Support Question
+    url: https://discourse.laminas.dev/c/questions/5
```
Comment (michalbundyra) [inline .github/ISSUE_TEMPLATE/config.yml]: Maybe we should have here link:
https://discourse.laminas.dev/c/questions/expressive/7

And for laminas we have:
https://discourse.laminas.dev/c/questions/components/6

Comment (froschdesign) [reply .github/ISSUE_TEMPLATE/config.yml]: Good point.

This brings a new task: update the forum. (I will ask @weierophinney if I can do that.)

Comment (Xerkus) [reply .github/ISSUE_TEMPLATE/config.yml]: I agree. But at this time I'm not sure those categories will remain. They might be just renamed or might be archived.

Comment (michalbundyra) [reply .github/ISSUE_TEMPLATE/config.yml]: I hope we can rename them and old links will be redirected, but not sure if it is possible?

Comment (Xerkus) [reply .github/ISSUE_TEMPLATE/config.yml]: @michalbundyra  Category text name does not matter, it is just for link readability. It is routed by id.

**Comment by froschdesign** [comment]
@michalbundyra 
> Not sure how it is solved by others (doctrine?)

I like the way from [Hugo](https://gohugo.io/). The Gitter chat contains this note at the top of the page:

> DEV DISCUSSIONS ONLY ... refer to https://gohugo.io for docs and https://discourse.gohugo.io for support and usage questions

And on the website they have 3 links:

* File an Issue (https://github.com/gohugoio/hugo/issues/new)
* Get Help (https://discourse.gohugo.io/)
* Discuss Source Code (https://gitter.im/spf13/hugo)

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (froschdesign) [reply .github/ISSUE_TEMPLATE/config.yml]: We have now:

* https://discourse.laminas.dev/c/questions/components/6
* https://discourse.laminas.dev/c/questions/mezzio/7
* https://discourse.laminas.dev/c/questions/api-tools/8

(See also: https://meta.discourse.org/t/how-to-redirect-a-category-using-permalink-rules/43380)

Comment (weierophinney) [reply .github/ISSUE_TEMPLATE/config.yml]: I agree that we should link to the specific category for the given project/subproject. @froschdesign has already renamed them (as he's noted in the comment prior to mine). I think with that change, we can get these merged against all three orgs.

**Pull request merged**